src/client.rs

```diff
@@ -16,13 +16,17 @@ use crate::replicon_core::{
     REPLICATION_CHANNEL_ID,
 };
 
+pub(crate) mod diagnostics;
+use diagnostics::{ReplicationStats, ReplicationStatsPlugin};
+
 pub struct ClientPlugin;
 
 impl Plugin for ClientPlugin {
     fn build(&self, app: &mut App) {
         app.add_plugins((RenetClientPlugin, NetcodeClientPlugin))
             .init_resource::<LastRepliconTick>()
             .init_resource::<ServerEntityMap>()
+            .add_plugins(ReplicationStatsPlugin)
```

Comment:
Looks like in Bevy plugins like this aren't enabled by default.
Maybe make this plugin public?
Also looks like Bevy use [`Diagnostics` in names](https://docs.rs/bevy/latest/bevy/diagnostic/index.html). So how about to rename it into `ReplicationDiagnosticsPlugin`?

Reply:
Actually, I would suggest `ClientDiagnosticsPlugin`. In the future we could have similar thing for server.